2025/selection-process.md

```diff
@@ -0,0 +1,81 @@
+
+
+## Timeline
+
+This is the general timeline for the Interop 2025 proposal selection process. The dates are centered on the Interop team meeting (Thursdays). Please note that the timelines are subject to change.
+
+*   Proposal submission window (3 weeks): Sept 19th, 2024 to Oct 3rd, 2024
+*   First round of feedback on proposals that do not meet the Interop project criteria (2 weeks): Oct 3rd, 2024 to Oct 17th, 2024
+*   Proposal selection and prioritization: Oct - Dec 2024
+*   Interop 2024 carryover evaluation and 2025 grouping: Dec 2024 - Jan 2025
+*   Scope of the Interop 2025 project will be published, the first half of February, 2025.
+
+
+## Proposal selection phases
+
+
+### Phase 1: Filtering (Oct 3rd to Oct 17th)
+
+Interop meeting dates: Oct 3rd, Oct 10th, Oct 17th
+
+[Visibility] Outcomes from this phase will be published on the Interop repo.
+
+In this stage, proposals that do not meet the criteria set by the Interop team will be filtered out. Feedback would be provided to the proposal authors to verify any additional data points that should be considered prior to exclusion. The Interop team will decide on the exclusion of proposals, based on consensus and proposal authors will be informed at the end of this phase. 
+
+There are two steps to this phase:
+1. Proposal refinement: Oct 4th, 2024 to Oct 10th, 2024 - feedback would be provided to proposal authors on the Interop repo. 
+2. First round of feedback on proposals that do not meet the Interop project criteria would be provided to proposal authors on the Interop repo.
+
+
+### Phase 2: Ranking (Oct 18th to Oct 31st)
+
+Interop meeting dates: Oct 24th, Oct 31st
+
+[Visibility] All decisions, made in Interop team meetings during this phase, will be confidential. However, organizations can choose to publish their own priorities, for the Interop proposals, outside of the Interop program.
+
+In this stage, the focus would be on organizations picking areas where there is internal alignment on the specific feature or area being important enough to work on from a user/web developer perspective. Each organization will have different rubrics for prioritization. However, where applicable, organizations can choose to have any data used for prioritization be shared publicly.
+
+3. Each organization will select proposals for the prioritization discussion by putting a High/Low priority signal against each proposal where they have internal alignment to do the work. A low priority signal would indicate a willingness to support the proposal, if other organizations also deem it to be important. Organizations can also express any objections at this point.
```

Comment:
Is there a "neutral" vote option implied here as well, or is that effectively what the "Low priority" vote means?

For example, if an organization chooses not to place a vote on a given proposal, is that interpreted as if they'd voted "Low priority"? Or is there a difference in how these are recorded such that voting "Low priority" is a slightly more positive signal than "abstain"?